lib/mastodon/cli/maintenance.rb

```diff
@@ -40,6 +40,10 @@ class Webhook < ApplicationRecord; end
     class BulkImport < ApplicationRecord; end
     class SoftwareUpdate < ApplicationRecord; end
 
+    class DomainBlock < ApplicationRecord
```

Comment:
Without this definition here, the "real" class was being used, and applying it's validations, so the update could not actually happen in this script.